### PR TITLE
Update RoninDojo project description and summary

### PR DESCRIPTION
**Pull request by lucasmogu**
Update "summary" and "description" as per OpenSats/volunteers#1

### CODE REVIEW

File: data/projects/ronin_dojo.mdx [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 title: 'RoninDojo'
 dateAdded: '2022-04-02'
-summary: 'We help people have a feature rich bitcoin experience, maintain privacy, and thrive as a sovereign individual.'
+summary: 'RoninDojo helps bitcoiners have a feature-rich bitcoin experience, maintain privacy, and thrive as sovereign individuals by developing easy-to-use hardware (such as the Tanto full bitcoin node) and software (RoninUI which uses TOR by default).'
```
Comment (Arvin21M) [inline data/projects/ronin_dojo.mdx:4]:
```suggestion
summary: 'RoninDojo helps Bitcoiners have a feature-rich bitcoin experience, maintain privacy, and thrive as sovereign individuals by developing easy-to-use hardware (such as the Tanto full Bitcoin node) and software (RoninUI which uses TOR by default).'
```

File: data/projects/ronin_dojo.mdx [hunk 2]
```diff
@@ -15,4 +15,4 @@ tags: ['Bitcoin', 'Privacy']
 
 ## About this project
 
-RoninDojo develops hardware and software to make it easy for anyone to use bitcoin privately as if it were cash. Some of our products include bitcoin nodes, steel backup plates, powerful software, premium support, and a lot more coming soon. We are working hard to increase accessibility for bitcoin users to connect their wallet to a node, use privacy enhancing coinjoin, do their own self custody, and so much more. Financial sovereignty is our goal and we won't slow down until it is achieved!
+RoninDojo products include Bitcoin nodes, steel backup plates, powerful storage, premium support, and much more coming soon. RoninDojo works hard to increase accessibility for bitcoiners to connect their wallet to a node, use privacy-enhancing coinjoin, and do self-custody.
```
Comment (Arvin21M) [inline data/projects/ronin_dojo.mdx:18]:
```suggestion
RoninDojo products include Bitcoin nodes, steel backup plates, powerful storage, premium support, and much more coming soon. RoninDojo works hard to increase accessibility for Bitcoiners to connect their wallet to a node, use privacy-enhancing CoinJoin, and do self-custody.
```

**Review by Arvin21M — CHANGES_REQUESTED** [review]
Just a few sprinkles of change.

File: data/projects/ronin_dojo.mdx [hunk 3]
```diff
@@ -15,4 +15,4 @@ tags: ['Bitcoin', 'Privacy']
 
 ## About this project
 
-RoninDojo develops hardware and software to make it easy for anyone to use bitcoin privately as if it were cash. Some of our products include bitcoin nodes, steel backup plates, powerful software, premium support, and a lot more coming soon. We are working hard to increase accessibility for bitcoin users to connect their wallet to a node, use privacy enhancing coinjoin, do their own self custody, and so much more. Financial sovereignty is our goal and we won't slow down until it is achieved!
+RoninDojo products include Bitcoin nodes, steel backup plates, powerful storage, premium support, and much more coming soon. RoninDojo works hard to increase accessibility for Bitcoiners to connect their wallet to a node, use privacy-enhancing CoinJoin, and do self-custody.
```
Comment (dergigi) [inline data/projects/ronin_dojo.mdx:18]: I don't think we should focus on the product side of this project, but on the fact that this is an open-source operating system that can run on many different devices: https://code.samourai.io/ronindojo/RoninOS

**Review by dergigi — CHANGES_REQUESTED** [review]

**Pull request closed without merging**